port_ocean/clients/port/mixins/entities.py

```diff
@@ -133,9 +134,11 @@ async def batch_delete_entities(
             return_exceptions=True,
         )
 
-    async def search_entities(self, user_agent_type: UserAgentType) -> list[Entity]:
-        query = {
-            "combinator": "and",
+    async def search_entities(
+        self, user_agent_type: UserAgentType, query: dict[Any, Any] | None = None
+    ) -> list[Entity]:
+        default_query = {
+            "combinator": "or",
```

Comment:
seems like the default search entities has changed from `and` to `or` is that on purpose, this has a huge performance impact on the query behavior and might result with different results

Reply:
oops...